tests/model/api/test_http_api_generator.py

```diff
@@ -95,6 +95,48 @@ def test_auth_iam_enabled(self):
             },
         )
 
+    def test_auth_iam_enabled_with_default_iam_authorizer(self):
```

Comment:
Why not a regular transform test? These are hard to read.

Reply:
Aren't these tests more stable (not flakey)?

Reply:
Transform tests aren't flaky either. Only some of the integ tests are flaky.